### DISCUSSION TITLE
Customizing default languages of the translation interface

### DISCUSSION THREAD

**Discussion by mwigman90** [directus/directus#2830] [Archive]
Currently, the translation interface has the following default languages:
`      "default": {
        "en": "English",
        "es": "Spanish",
        "de": "German",
        "fr": "French",
        "pt": "Portuguese",
        "zh": "Chinese",
        "ru": "Russian"
      },`

Is there a way to change these defaults? So when adding a new translation mapping, the correct languages for my website are already prefilled as translation options?

**Comment by rijkvanzanten** [comment]
When you create a translation field, you can set these to be whatever you want.

**Reply by mwigman90** [reply]
Right, but I would need to change them every time I create a translation field, since it will show the default languages for every new field. Currently I modified the .js file which contains the defaults for the translation interface, but that's not an ideal solution.
